### PR TITLE
Add remove cart button functionality

### PR DESCRIPTION
**Pull request by mwalsh01**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: client/store/cart.js [hunk 1]
```diff
@@ -88,7 +89,7 @@ export const cart = () => dispatch => {
 const reducer = (state = initialState, action) => {
   switch (action.type) {
     case SHOW_CART:
-      return state.cart
+      return {...state.cart}
```
Comment (ejcurtis) [inline client/store/cart.js]: {...state, cart: [...cart]

File: client/store/product.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ const getProducts = products => ({type: GET_ALL_PRODUCTS, products})
 
 export const product = productId => async dispatch => {
   try {
+    console.log('HERE HERE HERE')
```
Comment (ejcurtis) [inline client/store/product.js]: Delete uneeded console.log

File: server/api/products.js [hunk 3]
```diff
@@ -2,20 +2,21 @@ const router = require('express').Router()
 const {Product} = require('../db/models')
 module.exports = router
 
-router.get('/', async (req, res, next) => {
+router.get('/:id', async (req, res, next) => {
   try {
-    const products = await Product.findAll()
-    res.json(products)
+    console.log('AXIOS ID IN: ', req.params.id)
```
Comment (ejcurtis) [inline server/api/products.js]: delete console.log

File: client/store/cart.js (hunk 1 above)
Comment (mwalsh01) [reply client/store/cart.js]: I will make the change, but I'm not certain how this works. Looking to review this section to fix the cart.

**Pull request merged**